src/Symfony/Component/VarDumper/Cloner/Stub.php

```diff
@@ -23,6 +23,7 @@ class Stub
     const TYPE_ARRAY = 3;
     const TYPE_OBJECT = 4;
     const TYPE_RESOURCE = 5;
+    const TYPE_IMAGE = 6;
```

Comment:
No need for a  new type here, no need to change the Data class either: Stub objects have attributes that are exposed to dumpers already. See how is work the for "href" attribute.

Reply:
this is a new type of value here. So using a new type seems to make sense.

Reply:
I don't think so. We're not going to add new constants for every content-type we might add in the future. This type is about the way the data is structured. Here, it's a string.
Then we can add an attribute to hint dumpers about the meaning of the string. And make them able to display nicer output than just dumping the string.